src/styles.css

```diff
@@ -121,14 +121,14 @@ h2 {
 .ingredients-cost-section {
   display: flex;
   flex-direction: column;
-  width: 20%;
+  width: 18%;
   margin: 0px 0px 30px 30px;
   border: 1px solid #1B3231;
   padding: 0px 15px 0px 15px;
 }
 
 .instructions-section {
-  width: 80%;
+  width: 78%;
```

Comment:
Circle back to this. This was looking great earlier today. We must have tweaked something because now you have to scroll side-to-side to see the entire width of the page.